src/test/cloudWatchLogs/commands/searchLogGroup.test.ts

```diff
@@ -45,4 +45,23 @@ describe('searchLogGroup', async function () {
         accept(testInput)
         assert.strictEqual(await result, testInput)
     })
+
+    it('wizaard prioritizes logGroupInfo if passed in', async function () {
+        const fakeLogGroupNode = {
+            regionCode: 'test-region',
+            logGroup: {
+                logGroupName: 'test',
+            },
+        } as LogGroupNode
+
+        assert(fakeLogGroupNode.logGroup.logGroupName) // to avoid linting error
```

Comment:
Why bother with the fake node? Just pass the params directly.

Reply:
I was originally having it take the logGroupNode itself, but realized that it was overcomplicated and only needed this info.